seqr/utils/communication_utils.py

```diff
@@ -41,3 +43,28 @@ def send_welcome_email(user, referrer):
         password_token=user.password,
     )
     user.email_user('Set up your seqr account', email_content, fail_silently=False)
+
+
+def send_load_data_email(user, guid, namespace, name, individual_ids):
+    for data_manager in User.objects.filter(is_staff = True):
+        email_content = """
+        Hi there {full_name}--
+    
+        Collaborator project manager {user_name}/email{email} has created a new project with a GUID:
+          {guid}
+        from an AnVIL workspace {namespace}/{name}.
+
+        The new project is ready for loading. The created individual IDs are in attached file.
+    
+        Thanks!
+        """.format(
+            full_name = data_manager.get_full_name(),
+            user_name = user.get_full_name(),
+            email = user.email,
+            guid = guid,
+            namespace = namespace,
+            name = name,
+        )
+        mail = EmailMessage('Requesting loading AnVIL workspace data', email_content, DEFAULT_FROM_EMAIL, [data_manager.email])
```

Comment:
The subject line should be "AnVIL data loading request". You don't need to specify that you are using the default from email, django will default that for you so you should just leave out that argument unless you are overriding it. Also, if you could use keyword arguments here this would be more readable (so `EmailMessage(subject='AnVIL data loading request', body=email_content, to=recipients, attachments=[(filename, file_content)])`)